tools/appsre-worker-packer-container.sh

```diff
@@ -19,4 +19,4 @@ image_users = [$PACKER_IMAGE_USERS]
 EOF2
 fi
 
-/usr/bin/packer build "$PACKER_ONLY_EXCEPT" /osbuild-composer/templates/packer
+#/usr/bin/packer build "$PACKER_ONLY_EXCEPT" /osbuild-composer/templates/packer
```

Comment:
Can we keep building x86_64; just disable aarch for now?

Reply:
done